docs/index.rst

```diff
@@ -1,14 +1,125 @@
-Unified Workflow
-================
-
-TBD
+.. include:: sections/links.rst
 
 .. toctree::
    :hidden:
 
    sections/user_guide/index
    sections/contributor_guide/index
 
+
+Unified Workflow
+================
+
+``uwtools`` is an open-source modern Python package that helps automate common tasks needed for many standard numerical weather prediction (NWP) workflows and wraps Unified Forecast System (UFS) components in a highly configurable run-time interface (a driver) that maximizes flexibility, interoperability, and usability for any UFS Application.
```

Comment:
I'm definitely on board with renaming the repo to `uwtools` to match the package, and with renaming this page to `Unified Workflow Tools`.

I am still on the fence for the CLI. I have grown to like `uw` and I keep seeing `uwt` and pronouncing it like it rhymes with "newt".  I suppose `uwtools` would work there, too, but there might be some weird auto-complete awfulness that happens since we have several other things named the same. I understand that will happen with `uw` and `uwt`, too, but they're so short that perhaps it gets avoided more frequently. I'm just being weird and picky here, though. I think of all of them, though, I'm leaning toward `uwtools` since that is both consistent with the rest of it, and consistent with how other tools handle naming, e.g. `pylint`, `black`, `pytest`, etc.

If we're going to change, before the next release is probably the easiest way to go.

Reply:
I think leaving the CLI as `uw` is fine and a good compromise if we make other changes for consistency. I also like `uwtools`. (I threw out `uwt` to have something to _not_ like.) How about we think it over until we're back from the break and then decide? It will be a fairly easy change in any case. I'll set myself a reminder to bring it up again.

I think we _should_ rename the repo, though that might be the trickiest bit. Internal references should be easy to fix, but I need to make sure I can re-point Read the Docs at the renamed repo, and we'll all need to re-clone. Should be ok. Our Anaconda packages already use the `uwtools` name, so that's fine. Can you think of any other external references that we might need to update?

You could change the page title in this PR.

Reply:
Only all of my presentations to date and @elcarpenter, @NaureenBharwaniNOAA, and @WeirAE AMS content. ;-)

The good news is (I think) GitHub will keep the link to the old name in place until it's replaced by a new repo with that name in case we miss any.

Reply:
Oh shoot! The Read the Docs URL might also want to be included in this party.